Fixes #397 (#400)
unitaryfund/mitiq: CONTRIBUTING.md

If you add new features to a function or class, it is required to add tests for such object. Mitiq uses a nested structure for packaging tests in directories named `tests` at the same level of each module.

### Updating the documentation

> Follow the guidelines in the Contributing to docs [instructions](https://mitiq.readthedocs.io/en/latest/README.html) (look here on [GitHub](https://github.com/unitaryfund/mitiq/blob/master/README.md)), which include guidelines about updating the API-doc list of modules and writing examples in the users guide.

Reviewer: ```suggestion
Follow these [instructions for contributing to the documentation](https://mitiq.readthedocs.io/en/latest/contributing_docs.html) which include guidelines about updating the API-doc list of modules and writing examples in the users guide.
```
